hub/scenario/shorthands/create.tex

```diff
@@ -83,15 +83,20 @@
 		\end{array} \right]
 	\end{array} \right.
 \]
-and
+and the following two equivalent shorthands which we nonetheless give two different names
 \[
-	\scenCreateComputeAddress_{i} \define
-	\left[ \begin{array}{l}
-		% + \scenCreateException       _{i}  \\
-		% + \scenCreateAbort           _{i}  \\
-		+ \scenCreateFCond           _{i}  \\
-		+ \scenCreateExecution       _{i}  \\
-	\end{array} \right]
+	\left\{ \begin{array}{lcl}
+		\scenCreateComputeAddress _{i} & \define &
+		\left[ \begin{array}{l}
+			+ \scenCreateFCond     _{i} \\
+			+ \scenCreateExecution _{i} \\
+		\end{array} \right] \vspace{2mm} \\
+		\scenCreateLoadCreatee _{i} & \define &
+		\left[ \begin{array}{l}
+			+ \scenCreateFCond     _{i} \\
+			+ \scenCreateExecution _{i} \\
+		\end{array} \right] \\
+	\end{array} \right.
```

Comment:
Definition of the new shorthand. It's equivalent to an existing one with a different name. Semantically these things are different, whence the new shorthand.